### PR TITLE
Create a new category called '.NET Migrations' and move all related migration rules into it

### PR DESCRIPTION
**Pull request by YazhiChen**
Create a new category called '.NET Migrations' and move all related migration rules into it

### CODE REVIEW

File: categories/net-migrations/index.md
Comment (bradystroud) [inline categories/net-migrations/index.md:1]: @lukecookssw @YazhiChen 

I dont think we need a new top category for this, it falls under software development.

If we did have a top category, maybe it should be for Migrations in general (not limited to .NET)

1. Delete it if you agree

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: categories/net-migrations/index.md
Comment (tiagov8) [reply categories/net-migrations/index.md:1]: +1 top categories should be kept lean

Comment (YazhiChen) [reply categories/net-migrations/index.md:1]: This is required by @adamcogan to create a separate .NET Migration category. 
But detailed category still need to be reviewed and discussed with @lukecookssw

Comment (tiagov8) [reply categories/net-migrations/index.md:1]: Hi @YazhiChen , 

as per our conversation on Teams, the requirement is for a new category, not a new top category.

1. Please revert the new top category
2. Add a new **Rules to Better .NET Migrations** (no version to be added here, keep it generic) under "Software Development"

Comment (tiagov8) [reply categories/net-migrations/index.md:1]: Hi @YazhiChen 

As per my conversation with @lukecookssw 

I removed the part to not have version numbers. Please include them in your category titles so we can filter different migrations

**Review by tiagov8 — DISMISSED** [review]
1. Please action my comment https://github.com/SSWConsulting/SSW.Rules.Content/pull/6580#discussion_r1320033153

File: categories/net-migrations/index.md
Comment (YazhiChen) [reply categories/net-migrations/index.md:1]: Done. @tiagov8  @lukecookssw

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**